txpipe/maps.py

```diff
@@ -461,6 +464,17 @@ def data_iterator(self):
         # the source and lens map columns
         print("TODO: no lens weights here")
 
+        with self.open_input("photometry_catalog") as f:
```

Comment:
The docstring for TXMainMaps says this is faster than running TXSourceMaps and then TXLensMaps. Is this still true with this check in place?

Reply:
It should still be true yes, but the difference is minor and I've adjusted the docstring accordingly.